code/modules/mob/living/simple_animal/hostile/terror_spiders/ghost.dm

```diff
@@ -0,0 +1,43 @@
+
+/mob/living/simple_animal/hostile/poison/terror_spider/Topic(href, href_list)
+	if(href_list["activate"])
+		var/mob/dead/observer/ghost = usr
+		if(istype(ghost))
+			humanize_spider(ghost)
+
+/mob/living/simple_animal/hostile/poison/terror_spider/attack_ghost(mob/user)
+	humanize_spider(user)
+
+/mob/living/simple_animal/hostile/poison/terror_spider/proc/humanize_spider(mob/user)
+	if(key)//Someone is in it
+		return
+	var/error_on_humanize = ""
+	var/humanize_prompt = "Take direct control of [src]?"
+	humanize_prompt += "Role: [spider_role_summary]"
+	if(user.ckey in ts_ckey_blacklist)
+		error_on_humanize = "You are not able to control any terror spider this round."
+	else if(!ai_playercontrol_allowingeneral)
+		error_on_humanize = "Terror spiders cannot currently be player-controlled."
+	else if(spider_awaymission)
+		error_on_humanize = "Terror spiders that are part of an away mission cannot be controlled by ghosts."
+	else if(!ai_playercontrol_allowtype)
+		error_on_humanize = "This specific type of terror spider is not player-controllable."
+	else if(stat == DEAD)
+		error_on_humanize = "Dead spiders are not player-controllable."
+	if(jobban_isbanned(user, "Syndicate") || jobban_isbanned(user, "alien"))
+		to_chat(user,"You are jobbanned from role of syndicate and/or alien lifeform.")
```

Comment:
I think it is better, as well as simpler, to use Syndicate||Alien job bans, rather than a new ban type. I can't think of any case where I'd want someone job banned from Syndicate or Alien to be able to play these. Nor do I think that many people would be job banned from these specifically if they had their own ban type that wasn't used for anything else.